UI/xdg-data/com.obsproject.Studio.appdata.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>com.obsproject.Studio</id>
```

Comment:
I'm not sure should I change this line.

Reply:
No, please read the spec:

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Reply:
Thank you.